db/schema.rb

```diff
@@ -277,7 +277,7 @@
     t.string "hero_image"
     t.string "hero_image_credit"
     t.string "footer_logo"
-    t.string "tagline", default: "The Community Calendar"
```

Comment:
Confirming - we are removing this because we now have a default description that gets used as the tagline if the tagline does not exist. Is that right?